e2e_test/udf/tests/language_specific/javascript_async.slt

```diff
@@ -22,7 +22,7 @@ export async function fetch_todo(id) {
 }
 $$ WITH (async = true);
 
-query I
+query I retry 3 backoff 5s
```

Comment:
Maybe cdnjs or jsdelivr is more reliable than dummyjson?

Reply:
It may also because of some issue of our CI env I suppose. We can change if it still show up later.